code/modules/projectiles/projectile/beams.dm

```diff
@@ -46,25 +46,31 @@ In pvp they also have more lasting damages, such as infections, and pain form bu
 
 /obj/item/projectile/beam/weak/pistol_35
 	damage_types = list(BURN = 12)
+	armor_penetration = 5
 	recoil = 1
 
 /obj/item/projectile/beam/weak/light_rifle_257
-	damage_types = list(BURN = 13)
+	damage_types = list(BURN = 18)
+	armor_penetration = 15
```

Comment:
Same AP as the parrent but why more damage?

Reply:
> Same AP as the parrent
To make it more readable
> why more damage
The damage value was taken from bullet/light_rifle_257/scrap with standard dealing 22 damage.
A deciding factor about taking ammo is how much damage you can do with each bullet to conserve on inventory space and ammo use. With such low values you would be burning through both space and materials